jigglypuffRL/deeprl/ppo1/ppo1.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.optim as opt
+from torch.autograd import Variable
+from torch.distributions import Categorical
```

Comment:
Will this always be Categorical? It can be other distributions as well right? (not sure)

Reply:
Sorry, forgot to remove this from the `ppo1` file. Have moved all the distribution stuff to the policy class